e2e/src/main/java/org/jetbrains/bsp/bazel/base/BazelBspTestScenarioStep.kt

```diff
@@ -0,0 +1,23 @@
+package org.jetbrains.bsp.bazel.base
+
+import io.vavr.CheckedRunnable
+import io.vavr.control.Try
+import org.apache.logging.log4j.LogManager
+import org.apache.logging.log4j.Logger
+
+class BazelBspTestScenarioStep(private val testName: String, private val testkitCall: CheckedRunnable) {
+    fun executeAndReturnResult(): Boolean {
+        LOGGER.info("Executing \"{}\"...", testName)
```

Comment:
```suggestion
        LOGGER.info { "Executing \"$testName\"..." }
```

and u can do similar thing in each logger occurrence in edited files